config/default.go

```diff
@@ -44,6 +44,7 @@ GenerateAggchainProofTimeout = "1h"
 [L2RPC]
 	Mode = "basic"
 	URL = "{{L2URL}}"
+	RetryMode = "backoff"
```

Comment:
maybe not now
but i feel that providing too many options to users complicates stuff, code and config which we were thinking of simplifying
like i am not sure how many people will be configuring retry mode with different backoffs etc

Reply:
ya but if we are exposing some config which maybe not that imp, we need to maintain the code as well
dont have a strong opinion but majorly in favour of not exposing these param and hardcode what we think make sense